### ISSUE TITLE
`rc mount/mount` mount helper error: fusermount: unknown option 'fsname'

### ISSUE THREAD

**Issue by putty182** [rclone/rclone#6044]
#### What is the problem you are having with rclone?

`rc mount/mount` has started failing in 1.58 with the following command:

````
rclone rc mount/mount fs=media: mountPoint=/data/mount/media
````

Log output (stdout + json)

````
2022/03/19 11:08:26 mount helper error: fusermount: unknown option 'fsname'
2022/03/19 11:08:26 mount FAILED: fusermount: exit status 1
{"level":"error","msg":"rc: \"mount/mount\": error: fusermount: exit status 1","source":"rcserver/rcserver.go:176","time":"2022-03-19T11:08:26.654392+00:00"}
````

#### Workaround

Adding `mountOpt='{ "DeviceName": "blah" }'` seems to workaround the issue (guessed from commit [f4c40bf7](https://github.com/rclone/rclone/commit/f4c40bf79ddfd17c1d3c78bc856b346c3d393835))

````
rclone rc mount/mount fs=media: mountPoint=/data/mount/media mountOpt='{ "DeviceName": "rclone-media" }'
````

#### What is your rclone version (output from `rclone version`)

Docker ([sha256-505d65fa6a12](https://hub.docker.com/layers/rclone/rclone/latest/images/sha256-505d65fa6a12b81159f4d2d1c044d026df7d7a379121b87489bb62cd97bc213c))

````
rclone v1.58.0
- os/version: alpine 3.15.1 (64 bit)
- os/kernel: 5.4.0-1054-xxx (x86_64)
- os/type: linux
- os/arch: amd64
- go/version: go1.18
- go/linking: static
- go/tags: none
````

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
Thanks for pointing that out. It shouldn't be too hard to fix hopefully!

**Comment by ncw** [comment]
That was rather harder to fix than I thought!

Can you give this a try

[v1.59.0-beta.6081.045c37b65.fix-6044-mount-fsname](https://beta.rclone.org/branch/fix-6044-mount-fsname/v1.59.0-beta.6081.045c37b65.fix-6044-mount-fsname/) on branch [fix-6044-mount-fsname](https://github.com/rclone/rclone/tree/fix-6044-mount-fsname) (uploaded in 15-30 mins)
